helpers/tbans_helper.py

```diff
@@ -0,0 +1,45 @@
+from protorpc import transport
+
+from consts.client_type import ClientType
+from tbans_main import TBANSService
+
+from tbans.models.messages.messages import FCM, Webhook
+
+
+class TBANSHelper(object):
+
+    """
+    Helper class for sending push notifications.
+    Methods here should call the TBANS API to dispatch/queue notifications
+    """
+
+    @staticmethod
+    def _create_service():
+        return TBANSService.Stub(transport.HttpTransport('https://tbatv-prod-hrd.appspot.com/tbans/tbans'))
```

Comment:
Is there a way we can grab this domain/host (`tbatv-prod-hrd.appspot.com`) dynamically? The idea is that we want this to work seamlessly in a dev container, but also in other people's prod instances